src/App.js

```diff
@@ -13,7 +13,7 @@ const App = () => (
           <Types />
         </Route>
         <Route exact path="/">
-          <ExamplePage />
+          <HomePage />
         </Route>
         <Route>Not found</Route>
       </Switch>
```

Comment:
For now, could you add a new route for your MapPage? Don't integrate into the tabs for now, still working out some details there.